ddtrace/llmobs/_llmobs.py

```diff
@@ -75,30 +97,127 @@ def _stop_service(self) -> None:
             log.warning("Failed to shutdown tracer", exc_info=True)
 
     @classmethod
-    def enable(cls, tracer=None):
+    def _toggle(cls, switch: bool) -> None:
+        # _toggle is a helper function to ensure the state of config._llmobs_enabled is in sync with the class attribute
+        # since our integrations depend on config._llmobs_enabled to set llmobs data
+        config._llmobs_enabled = switch
+        cls.enabled = switch
+
+    @classmethod
+    def enable(
+        cls,
+        ml_app: Optional[str] = None,
+        integrations: Optional[List[str]] = None,
+        dd_apm_enabled=True,
```

Comment:
thoughts on this name? @Yun-Kim @sabrenner

Reply:
I think it could be confusing to have this config option named `dd_apm_enabled` while the "corresponding" env var is `DD_LLMOBS_NO_APM`. It feels like if both have the same value, they should mean the same thing, whereas now I think if both are set to `True` they mean opposite things.

Reply:
renaming to `dd_llmobs_no_apm` for consistency with env var and also going to update the enable() function so that all the env vars non-APM customers need to set are switched to the correct value when `dd_llmobs_no_apm` is true